src/theme/NavbarItem/DropdownNavbarItem.js

```diff
@@ -0,0 +1,180 @@
+                    fontWeight: 400,
+                    fontSize: "16px",
+                    lineHeight: "21px",
+                    textAlign: "center",
+                }}
+                aria-haspopup="true"
+                aria-expanded={showDropdown}
+                role="button"
+                href={props.to ? undefined : "#"}
+                className={clsx("navbar__link", className)}
+                {...props}
+                onClick={props.to ? undefined : (e) => e.preventDefault()}
+                onKeyDown={(e) => {
+                    if (e.key === "Enter") {
+                        e.preventDefault();
+                        setShowDropdown(!showDropdown);
+                    }
+                }}
+            >
+                {props.children ?? props.label}
+            </NavbarNavLink>
+            <ul className="dropdown__menu">
+                {items.map((childItemProps, i) => (
+                    <NavbarItem
+                        isDropdownItem
+                        onKeyDown={(e) => {
+                            if (i === items.length - 1 && e.key === "Tab") {
+                                e.preventDefault();
+                                setShowDropdown(false);
+                                const nextNavbarItem =
+                                    dropdownRef.current.nextElementSibling;
+                                if (nextNavbarItem) {
+                                    const targetItem =
+                                        nextNavbarItem instanceof
+                                        HTMLAnchorElement
+                                            ? nextNavbarItem
+                                            : // Next item is another dropdown; focus on the inner
+                                              // anchor element instead so there's outline
+                                              nextNavbarItem.querySelector("a");
+                                    targetItem.focus();
+                                }
+                            }
+                        }}
+                        activeClassName="dropdown__link--active"
+                        {...childItemProps}
+                        key={i}
+                    />
+                ))}
+            </ul>
+        </div>
+    );
+}
+function DropdownNavbarItemMobile({
+    items,
+    className,
+    position, // Need to destructure position from props so that it doesn't get passed on.
+    onClick,
+    ...props
+}) {
+    const localPathname = useLocalPathname();
+    const containsActive = containsActiveItems(items, localPathname);
+    const { collapsed, toggleCollapsed, setCollapsed } = useCollapsible({
+        initialState: () => !containsActive,
+    });
+    // Expand/collapse if any item active after a navigation
+    useEffect(() => {
+        if (containsActive) {
+            setCollapsed(!containsActive);
+        }
+    }, [localPathname, containsActive, setCollapsed]);
```

Comment:
I think `localPathname` is useless in here

Reply:
on the mobile and after localPathname changed, the active item needs to be updated